post_installation_configuration/bare-metal-configuration.adoc

```diff
@@ -6,13 +6,9 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-When deploying {product-title} on bare metal hosts, there are times when you need to make changes to the host either before or after provisioning. This can include inspecting the host's hardware, firmware, and BIOS details. It can also include formatting disks or changing modifiable BIOS settings. There are two resources that you can use with the Bare Metal Operator (BMO):
-
-* `BareMetalHost`
-* `HostFirmwareSettings`
-
-There is also a read-only `FirmwareSchema` resource, which you can use to determine the valid values that you can send to a host when making changes to host firmware settings.
+When deploying {product-title} on bare metal hosts, there are times when you need to make changes to the host either before or after provisioning. This can include inspecting the host's hardware, firmware, and BIOS details. It can also include formatting disks or changing modifiable BIOS settings. 
```

Comment:
You use "BIOS settings" here but "firmware settings" in the new document. BIOS settings is upstream Ironic terminology (that I personally regret, but still).

Reply:
I have updated this to firmware. I think firmware should be used pretty much everywhere now? I know there was some feedback recently to our team that BIOS is a term that belongs in the 1990s! :-p

Reply:
Yeah, we regret using BIOS there.. but you know, naming is hard. We had a room full of people with opinions, and BIOS end up the only thing everyone agreed to agree on.